experimental/GroebnerWalk/test/standard_walk.jl

```diff
@@ -0,0 +1,23 @@
+R1, (x,y,z) = polynomial_ring(QQ, ["x", "y", "z"])
```

Comment:
Could you move all of these definitiona into your `@testset`? Background to this is that `@testset` introduced a new scope, and without this change you pollute the global namespace, and thus maybe change results of tests run after your file